modules/fedimint-ln-server/src/lib.rs

```diff
@@ -753,18 +753,18 @@ impl ServerModule for Lightning {
         vec![
             api_endpoint! {
                 "/account",
-                async |module: &Lightning, dbtx, contract_id: ContractId| -> ContractAccount {
+                async |module: &Lightning, context, contract_id: ContractId| -> ContractAccount {
```

Comment:
lets use `ctx` instead of `context`

Reply:
What's the advantage of abbreviating?  It's not that long a name and I couldn't easily guess `ctx = context`.

Reply:
fair enough, I am biased. when I worked in rust gui, we called it ctx or even cx, but gui has way too many contexts.